README.md

```diff
@@ -322,3 +373,16 @@ npm publish
 ```
 
 For more information on publishing to npm, see the [npm publish documentation](https://docs.npmjs.com/cli/publish).
+
+## Third Party Software in `eleventy-plugin-fluid`
+
+Trivet is based on other publicly available software, categorized by license:
```

Comment:
I'm assuming this is a copy-and-paste error and should be `eleventy-plugin-fluid` instead of `Trivet`.